models/labels/cex/labels_cex_ethereum.sql

```diff
@@ -0,0 +1,496 @@
+{{config(alias='cex_ethereum')}}
+
+SELECT blockchain, address, name, category, contributor, source, created_at, updated_at
+FROM (VALUES
+    -- Binance, Source: https://etherscan.io/accounts/label/binance
+    (array('ethereum'),'0x3f5ce5fbfe3e9af3971dd833d26ba9b5c936f0be', 'Binance 1', 'cex', 'hildobby', 'static', timestamp('2022-08-28'), now())
```

Comment:
shouldn't `updated_at` be the same as `created_at`

Reply:
i could see it either way, but if they're the same, then we should just have ```updated_at``` only and leave out ```created_at```

Reply:
The idea here is to have created_at reflect the moment at which contributors create the query/static labels for the first time, while updated_at reflects the last date at which the table was updated/rebuilt